packages/manager/modules/bm-server-components/src/order-private-bandwidth/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,106 @@
+{
+  "server_error_bandwidth_order": "Une erreur est survenue lors de la commande de bande passante : {{error}}",
+  "server_order_bandwidth_title": "Commander de la bande passante",
+  "server_order_bandwidth_vrack_title": "Commander de la bande passante",
+  "server_order_bandwidth_not_available": "Aucun type de bande passante n'est commandable sur le serveur {{t0}}",
+  "server_order_bandwidth_help": "Les offres Premium, Platinum et Ultimate garantissent la qualité de la bande passante sur une zone géographique plus ou moins étendue jusqu'au plus près de vos clients.",
+  "server_order_bandwidth_ready": "Votre commande est prête. Sous réserve de paiement, votre bande passante devrait être disponible prochainement.",
+  "server_order_bandwidth_see_order": "Voir le bon de commande",
+  "server_order_bandwidth_vrack_load": "Chargement des offres en cours...",
+  "server_order_bandwidth_empty_plans": "Impossible de trouver des plans pour mettre à niveau",
+  "select_bandwidth_type_label": "Sélectionnez une offre",
+  "select_bandwidth_vrack_type_label": "Sélectionnez une offre",
+  "select_bandwidth_plan_summary": "Récapitulatif de la commande",
+  "server_order_bandwidth_vrack_plan_text": "de bande passante privée, trafic illimité (vRack)",
+  "server_order_bandwidth_declaration": "La date de renouvellement de votre option sera alignée avec celle de votre serveur. Le montant affiché correspond au premier mois de facturation de votre option, calculé au prorata temporis du mois en cours.",
+  "server_order_bandwidth_info": "Le montant indiqué correspond au montant facturé au prochain renouvellement de votre option.",
+  "server_order_bandwidth_warning": "La modification de votre bande passante sera appliquée quelques minutes après l’activation de l’option.",
+  "server_order_bandwidth_auto_pay": "Payer avec le moyen de paiement par défaut",
+  "plan_vrack-bandwidth-1000-option": "1 Gbps de bande passante privée garantie - Trafic illimité - Option vRack",
+  "plan_vrack-bandwidth-3000-option": "3 Gbps de bande passante privée garantie - Trafic illimité - Option vRack",
+  "plan_vrack-bandwidth-1000-option-infra": "1 Gbps de bande passante privée garantie - Trafic illimité - Option vRack",
+  "plan_vrack-bandwidth-3000-option-infra": "3 Gbps de bande passante privée garantie - Trafic illimité - Option vRack",
+  "plan_vrack-bandwidth-4000-option-infra": "4 Gbps de bande passante privée garantie - Trafic illimité - Option vRack",
+  "plan_vrack-bandwidth-5000-option-infra": "5 Gbps de bande passante privée garantie - Trafic illimité - Option vRack",
```

Comment:
is't easly duable to have this :
`plan_vrack-bandwidth-option-infra: "{{ bandwith }} de bande passante privée garantie - Trafic illimité - Option vRack"`

instead of one entry for each bandwidth option.

Reply:
Good suggestion. This is legacy code. I have just moved/split it to new file and module. The aim of the refactoring is to do with minimal code changes. This can be taken as a separate task.